src/models/storage.ts

```diff
@@ -114,6 +114,25 @@ export class BrowserStorage {
     }
     return;
   }
+
+  // Use for Chrome only.
+  // https://github.com/Authenticator-Extension/Authenticator/issues/412
+  static async clearLogs() {
```

Comment:
async doesn't work as expected as we use callback in the function. I found this project, and it seems a good option for us: https://github.com/Lusito/webextension-polyfill-ts

Reply:
I'm only using the async to make `await this.getStorageLocation` work properly. Moving to that polyfill should be another PR entirely since it would change so much.

Reply:
+ wouldn't this not work anywhere we have chrome specific calls?

Reply:
It seems work with Chrome: https://github.com/mozilla/webextension-polyfill#supported-browsers

Reply:
fixed async issue. I agree that we should move to webextension-polyfill.